adds kubecon NA 2021 roundup blog (#842)
pulumi/pulumi-hugo: themes/default/content/blog/kubecon-na-2021-roundup/index.md

---
title: "Kubecon NA 2021 Roundup"

date: 2021-11-08T22:50:38Z

draft: false

meta_desc: Kat Cosgrove's recording roundup of the trends, updates, and gems from Kubecon NA 2021!

meta_image: meta.jpg

authors:
    - kat-cosgrove

tags:
    - kubecon
    - kubernetes
    - conferences
---

> Kubecon North America is over, but the recordings are now online! Every talk you wanted to see and couldn’t is available on YouTube, so here’s some highlights -- cloud native trends, updates from projects and SIGs, and a few of my favorite talks!

Reviewer: ```suggestion
KubeCon North America 2021 is over, but the recordings are now online! Every talk you wanted to attend and couldn’t is available on YouTube, so here’s some highlights&mdash;cloud native trends, updates from projects and SIGs, and a few of my favorite talks!
```

<!--more-->

> Kubecon North America is over, but the recordings are now online! Every talk you wanted to see and couldn’t is available on YouTube, so here’s some highlights from me -- cloud native trends, updates from projects and SIGs, and a few of my favorite talks!

Reviewer: Was this line supposed to be duplicated?

#### SBOM is Coming: Why You Should Care and How You Can Help - Frederick Kautz & Allan Friedman

{{< youtube "4wUejdZ6KHM?rel=0" >}}

> A major trend throughout the conference was supply chain security and SBOMs. There were quite a few talks on these subjects, but Frederick Kautz and Allan Friedman give a great overview of exactly what an SBOM is, and why it’s important.

Reviewer: ```suggestion
A major trend throughout the conference was supply chain security and software bills of materials (SBOMs). There were quite a few talks on these subjects, but Frederick Kautz and Allan Friedman give a great overview of exactly what an SBOM is, and why it’s important.
```

#### Notary: State of the Container Supply Chain - Justin Cormack, Docker & Steve Lasker, Microsoft

{{< youtube "hpcQJ21mn_g?rel=0" >}}

> On supply chain security, Steve Lasker presents updates to Notary v2. It’s changed quite a bit since v1 and there are more changes on the horizon. With the rising concerns surrounding supply chain security, I expect Notary and tools like it to be increasingly important. No worries if you aren’t already familiar with Notary -- Steve explains it well.

Reviewer: ```suggestion
On supply chain security, Steve Lasker presents updates to Notary v2. It’s changed quite a bit since v1 and there are more changes on the horizon. With the rising concerns surrounding supply chain security, I expect Notary and tools like it to be increasingly important. No worries if you aren’t already familiar with Notary&mdash;Steve explains it well.
```

#### A Vulnerable Tale About Burnout - Julia Simon, CloudOps

{{< youtube "lpiXbfOTNYw?rel=0" >}}

Taking things to a more human place, burnout was a common theme this year. It’s something that we don’t talk about as much as we should, but close to two years of isolation is forcing us all to confront it in ways we may not have expected. This talk by Julia Simon is important to watch not just so that you can learn to recognize these problems in yourself, but so that you can recognize them in your peers.

#### PodSecurityPolicy Replacement: Past, Present, and Future - Tabitha Sable & Tim Allclair

{{< youtube "HsRRmlTJpls?rel=0" >}}

> PodSecurityPolicy has been deprecated. Many of you may be relying on this, and deprecations frequently sound scary, but Tabitha Sable and Tim Allclair are here to soothe your fears. They go to great lengths explaining what PSP is, why it was deprecated, what exactly this means for people relying on it, and where we’re going next.

Reviewer: ```suggestion
PodSecurityPolicy (PSP) has been deprecated. Many of you may be relying on this, and deprecations frequently sound scary, but Tabitha Sable and Tim Allclair are here to soothe your fears. They go to great lengths explaining what PSP is, why it was deprecated, what exactly this means for people relying on it, and where we’re going next.
```

#### SIG Contributor Experience Deep Dive - Alison Dowdney, Weaveworks & Christoph Blecker, Red Hat

{{< youtube "QOiyWWFjG5Q?rel=0" >}}

> An open source project lives or dies by its contributors, both technical and non-technical. Kubernetes is huge and requires a lot of contributors to keep it growing, so we have SIG Contributor Experience to make sure they stay happy and the community stays healthy. Alison Dowdney and Cristoph Blecker go into just how SIG Contributor Experience operates, plus updates. If you’re in a leadership position for an open source project, there are some lessons to be learned from this talk.

Reviewer: ```suggestion
An open source project lives or dies by its contributors, both technical and non-technical. Kubernetes is huge and requires a lot of contributors to keep it growing, so we have SIG Contributor Experience to make sure they stay happy and the community stays healthy. Alison Dowdney and Christoph Blecker go into just how SIG Contributor Experience operates, plus updates. If you’re in a leadership position for an open source project, there are some lessons to be learned from this talk.
```
